package.json

```diff
@@ -41,7 +41,7 @@
     "browserify-transform"
   ],
   "dependencies": {
-    "brfs": "~1.0.2",
+    "brfs": "^1.4.3",
```

Comment:
Is this version bump really necessary? I had some problem using brfs vers greater then 1.0